File: Ahnhyunwoo/2022-03-28.cpp
#include <string>
#include <vector>
#include <math.h>
using namespace std;

vector<int> solution(int n, int s) {
    vector<int> answer;
    if(n > s) {
        answer.push_back(-1);
        return answer;
    }
    int num = s / n;
    int k = s % n;
    for(int i = k; i < n;i++) {
        answer.push_back(num);
    }
    for(int i = 0; i <k;i++) {
        answer.push_back(num + 1);
    }  
    return answer;
}

File: juhyunyu/20220328.java
// 프로그래머스
// 가운데 수 가져오기
class Solution {
    public String solution(String s) {
        String answer = "";
        int len = s.length();
        if(len % 2 != 0) {
            answer = s.substring((len-1)/2,(len-1)/2+1);
        }
        else {
            answer = s.substring((len-1)/2,(len-1)/2+2);                                                        
        }
        return answer;
    }
}
